recipes.js

```diff
@@ -5,6 +5,86 @@ const data = require('./data.js');
 mongoose.connect('mongodb://localhost/recipeApp')
   .then(() => {
     console.log('Connected to Mongo!');
-  }).catch(err => {
+    Recipe.insertMany([
```

Comment:
Aqui poderia ser algo como `Recipe.insertMany(data)` já que está importando os dados na variável `data` no início.